src/main/resources/com/google/api/codegen/ruby/rubocop.yml.snip

```diff
@@ -2,9 +2,8 @@
   AllCops:
     Exclude:
       - "{@metadata.identifier}.gemspec"
-      - "lib/{@metadata.protoPath}/**/*"
-      # This should be removed in the future after we are sanitizing client files.
-      - "lib/{@metadata.versionPath}/**/*"
+      # This should be removed in the future once GAPIC files conform more closely to rubocop checks.
```

Comment:
nit: are we fine having this be specific to the google namespace rather than the the generic first namespace of a package?

Reply:
Agreed that this should be generic, but this is also just a workaround to get CI to pass in google-cloud-ruby until #1536 and #1120 are fixed, so leaving as-is.